import "base/aperture_parameters.proto";

message OpticalVortexParameters {
  // The number of 2*PI cycles that are present in the vortex.
  optional double num_cycles = 1;
}

extend mats.ApertureParameters {
  optional OpticalVortexParameters optical_vortex_params = 15725;
}
